src/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsEventSourceLifetime.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using System.Diagnostics.Tracing;
+using System.Reflection;
+
+namespace BasicEventSourceTests
+{
+    public class TestsEventSourceLifetime
+    {
+        /// <summary>
+        /// Validates that the EventProvider AppDomain.ProcessExit handler does not keep the EventProvider instance
+        /// alive.
+        /// </summary>
+        [ActiveIssue(4871, PlatformID.AnyUnix)]
+        [ActiveIssue(4871, PlatformID.Linux)]
```

Comment:
Linux is included in AnyUnix, so you just need the line for AnyUnix and can delete the second.  Same goes for all of the other occurrences of this.